upgrade-2.x.sh

```diff
@@ -309,7 +309,7 @@ function _patch_supervisor_version() {
     UPDATER_SUPERVISOR_TAG="v${version}"
 
     # Get the supervisor id
-    resp=$(CURL_CA_BUNDLE="${TMPCRT}" ${CURL} --header "Authorization: Bearer ${APIKEY}" "${API_ENDPOINT}/v5/supervisor_release?\$select=id,image_name&\$filter=((device_type%20eq%20'$SLUG')%20and%20(supervisor_version%20eq%20'${UPDATER_SUPERVISOR_TAG}'))")
+    resp=$(CURL_CA_BUNDLE="${TMPCRT}" ${CURL} --header "Authorization: Bearer ${APIKEY}" "${API_ENDPOINT}/v5/supervisor_release?\$select=id,image_name&\$filter=device_type%20eq%20'$SLUG'%20and%20supervisor_version%20eq%20'${UPDATER_SUPERVISOR_TAG}'")
```

Comment:
This should should really be updated to v6 and the others to v7 but it was a more intrusive change so I skipped it for now

Reply:
Noted; I expect a lot of renovations in this code in the near future, which will be good time to implelement.